sites/teams.html

```diff
@@ -25,73 +25,73 @@ <h1 class="main-header">
 </nav>
 <section class="teams-selection-section">
     <div class="teams-selection">
-        <div class="team">
-            <div class="team__select team__select--country">
-                <div class="select__arrow">
+        <div class="teams-selection__team teams-selection__team--home">
+            <div class="teams-selections__team-select teams-selections__team-select--country">
+                <div class="teams-selection__select-arrow teams-selection__select-arrow--country-left">
                     <img src="../img/navigation/left-arrow.png" alt="left arrow">
                 </div>
-                <div class="select__wrapper select__wrapper--country">
-                    <div class="select__country-name">
+                <div class="teams-selection__select-wrapper teams-selection__select-wrapper--country">
+                    <div class="teams-selection__select-country--name">
                         <p>POLAND</p>
                     </div>
-                    <div class="select__country-image">
+                    <div class="teams-selection__select-country--image">
                         <img src="../img/flags/poland.png" alt="poland">
                     </div>
                 </div>
-                <div class="select__arrow">
+                <div class="teams-selection__select-arrow teams-selection__select-arrow--country-right">
                     <img src="../img/navigation/right-arrow.png" alt="right arrow">
                 </div>
             </div>
-            <div class="team__select team__select--team">
-                <div class="select__arrow">
+            <div class="teams-selections__team-select teams-selections__team-select--team">
+                <div class="teams-selection__select-arrow teams-selection__select-arrow--team-left">
                     <img src="../img/navigation/left-arrow.png" alt="left arrow">
                 </div>
-                <div class="select__wrapper select__wrapper--team">
-                    <div class="select__team-name">
+                <div class="teams-selection__select-wrapper teams-selection__select-wrapper--team">
+                    <div class="teams-selection__select-team--name">
                         <p>GÓRNIK</p>
                     </div>
-                    <div class="select__team-image">
+                    <div class="teams-selection__select-team--image">
```

Comment:
Modificator in BEM should be always use with base element, so teams-selection__image will more correct